scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -181,13 +181,13 @@ def main():
 
     python_utils.PRINT('Checking e2e tests are captured in .travis.yml...')
     protractor_test_suites = get_e2e_suite_names_from_protractor_file()
-    travis_e2e_jobs = get_e2e_suite_names_from_jobs_travis_yml_file()
+    travis_e2e_suites = get_e2e_suite_names_from_jobs_travis_yml_file()
```

Comment:
Should we change this test and make sure that all the flags are enabled i.e, they are set to true? (if not commented) [not sure!]

Reply:
mentioned this offline, but this is good to keep as it does a sanity check for the prefix of the suite name.

Reply:
Yeah but we are doing it bit differently now, Earlier this structure was useful as we were using the travis_e2e_jobs variable in other places but now the same structure creates confusion and I don't think we need that structure anymore!

Should we create a simple function `check_e2e_job_section_not_empty`? that way we can avoid having confusion with variable and function names. What do you think?

Reply:
I'll do it in a follow up PR since I won't be revisiting this until much later in tmw in the day. That being said, please do unlock the PR if there's no other pressing issues.

Reply:
Discussed this offline, marking this as resolved.